tests/unittests/bases/test_ddp.py

```diff
@@ -88,6 +88,7 @@ def _test_ddp_compositional_tensor(rank: int, worldsize: int = NUM_PROCESSES) ->
 
 @pytest.mark.DDP()
 @pytest.mark.skipif(sys.platform == "win32", reason="DDP not available on windows")
+@pytest.mark.skipif(not hasattr(pytest, "pool"), reason="DDP pool is not available.")
```

Comment:
Are these really needed?

Reply:
Yes and no. If you run `pytest` locally without the `-m` argument set to either DDP or not DDP you will get that all tests fail because the pool is not available.

Reply:
Ok, fair point, so any leap on crashing on test collection?